src/windows/ui/main-window.ui

```diff
@@ -1394,7 +1406,7 @@
     <string>Remove from Project</string>
    </property>
    <property name="toolTip">
-    <string>Remove from </string>
+    <string>Remove from Project</string>
```

Comment:
What is this? lol  I'm gonna look into that one.

Reply:
Yeah that was missing. Thought I'd fix it